library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttParser.java

```diff
@@ -60,6 +65,8 @@
   private static final String WEBVTT_CUE_SETTING_STRING = "\\S*:\\S*";
   private static final Pattern WEBVTT_CUE_SETTING = Pattern.compile(WEBVTT_CUE_SETTING_STRING);
 
+  private static final String WEBVTT_PERCENTAGE_NUMBER_STRING = "^([0-9]+|[0-9]+\\.[0-9]+)$";
```

Comment:
Fine to leave this one too, although wouldn't it be clearer and match the spec more directly if it were like: ```"^([0-9]+(\\.[0-9]+))$```